server/apps/impl/proxy.go

```diff
@@ -5,9 +5,44 @@ package impl
 
 import (
 	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/pkg/errors"
 )
 
+func (s *service) filterContext(c *apps.Call) error {
+	checkTeam := true
+	if c.Context.ChannelID != "" {
```

Comment:
We should also be checking for the root post id here, and using that to source the channel id if it is present.